dashboard/src/components/LoginForm/LoginForm.test.tsx

```diff
@@ -37,30 +34,23 @@ const defaultProps = {
 
 const authenticationError = "it's a trap";
 
-context("while authenticating", () => {
```

Comment:
did you miss this test? (We talked offline to avoid using `itBehavesLike`)

Reply:
You're absolutely right, thanks for letting me know. Adding it back.